newsroom/templates/email_layout.html

```diff
@@ -0,0 +1,5 @@
+<p>{{ name }},</p>
+
+{% block content %}{% endblock %}
+
+{% include "email_footer.html" %}
```

Comment:
now you can add the footer here instead of including it, or is it included anywhere else?

Reply:
🥇 <-- This is what I need I guess